source/jormungandr/jormungandr/street_network/asgard.py

```diff
@@ -227,38 +273,101 @@ def handle_car_no_park_modes(mode):
             return FallbackModes.car.name
         return mode
 
-    def _create_direct_path_request(
-        self, mode, pt_object_origin, pt_object_destination, fallback_extremity, request, language="en-US"
-    ):
-        # asgard and kraken have a part of request in common
-        req = create_kraken_direct_path_request(
-            self, mode, pt_object_origin, pt_object_destination, fallback_extremity, request, language
+    @staticmethod
+    def is_alternatives_request(mode, direct_path_type, request):
+        return (
+            mode == 'bike'
```

Comment:
Do we need to test for bikes too? From what I understand, it's sure is for GeoVelo only at the moment but who knows, we might have other mode with the same functionality. If you think it's better to keep it, then maybe rename the function into `is_bike_alternatives_request` to make it explicit about "it's only bike"?

Reply:
You understood it well. the alternative journeys are only available for bike mode for the moment, but not only bike for sure...